tests/phpunit/test-wc-stripe-payment-request.php

```diff
@@ -149,6 +149,10 @@ public function test_get_shipping_options_returns_chosen_option() {
 
 		$flat_rate              = $this->get_shipping_option( $this->flat_rate_id );
 		$expected_display_items = [
+			[
+				'label'  => 'Subtotal',
+				'amount' => 1000,
+			],
```

Comment:
This specific test is about checking Shipping Options but here in this PR we will also return the "Subtotal" as part of it.

To change the format of the resulting object of the `get_shipping_options`' method will require a significant refactor which is out of the scope of this PR.